build/webpack.js

```diff
@@ -258,14 +258,10 @@ export default async function getBaseWebpackConfig (dir: string, {dev = false, i
         // required not to cache removed files
         useHashIndex: false
       }),
-      new webpack.DefinePlugin({
-        'process.env.NODE_ENV': JSON.stringify(dev ? 'development' : 'production')
```

Comment:
Let's add a test for the behavior so that we're 100% sure process.env.NODE_ENV is replaced in the future 👍

Reply:
As a *lot* of optimizations are done based on it.

Reply:
The easiest way is fetching the js bundle on the client side 👍

Reply:
done